content/knowledge-base/guides/write-a-composition-function-in-go.md

```diff
@@ -0,0 +1,832 @@
+---
+title: Write a Composition Function in Go
+state: beta
+alphaVersion: "1.11"
+betaVersion: "1.14"
+weight: 80
+description: "Composition functions allow you to template resources using Go"
+---
+
+Composition functions (or just functions, for short) are custom programs that
+template Crossplane resources. Crossplane calls composition functions to
+determine what resources it should create when you create a composite resource
+(XR). Read the
+[concepts]({{<ref "../../master/concepts/composition-functions">}})
```

Comment:
@plumbis I'd prefer to link to latest, or failing that v1.14 specifically.

I tried a regular `[foo](/latest)` style link, but HTML test didn't like it.

Reply:
You can add `docs.crossplane.io/latest/*` to the htmltest config, since this is a deploy-time path. 

https://github.com/crossplane/docs/blob/5dcc5b814a0306331200507dc442aa65af867ac4/utils/htmltest/.htmltest.yml#L6